packages/schema/src/config/common.ts

```diff
@@ -20,6 +20,18 @@ export default defineUntypedSchema({
    */
   extends: null,
 
+  /**
+   * Specify a compatibility date for your app.
+   *
+   * This is used to control the behavior of presets in Nitro, Nuxt Image
+   * and other modules that may change behavior without a major version bump.
+   *
+   * We plan to improve the tooling around this feature in the future.
+   *
+   * @type {typeof import('compatx').DateString | Record<string, typeof import('compatx').DateString>}
```

Comment:
My bad forgot to mention we have `CompatibilityDateSpec` as well, it is string or object map from known platform keys to version we should used it

Reply:
~> https://github.com/nuxt/nuxt/pull/27521